phpstan-baseline.php

```diff
@@ -5470,7 +5470,7 @@
 $ignoreErrors[] = [
 	// identifier: empty.notAllowed
 	'message' => '#^Construct empty\\(\\) is not allowed\\. Use more strict comparison\\.$#',
-	'count' => 10,
+	'count' => 11,
```

Comment:
We should strive not to increase our baseline and fix things as they occur, unless it's a false positive.

Reply:
Done